doc/api/util.md

```diff
@@ -1425,8 +1425,10 @@ changes:
       `false`, values for the option are last-wins. **Default:** `false`.
     * `short` {string} A single character alias for the option.
     * `default` {string | boolean | string\[] | boolean\[]} The default option
-      value when it is not set by args. It must be of the same type as the
-      `type` property. When `multiple` is `true`, it must be an array.
+      value when this argument is not passed. It must be of the same type as the
+      `type` property. When `multiple` is `true`, it must be an array. For
+      `string` arguments, if the argument is passed it must still be followed by
```

Comment:
I get the intent, but see some possible confusions. The arguments can be implicit (so not passed as such), and a value can be omitted when `strict:false`.

However, a minor suggestion if run with this wording:

```suggestion
      `string` arguments, if the option is passed it must still be followed by
```

Reply:
Well, they're still passed by someone! Passed by CLI or passed explicitly, but passed by someone.